assets/wizards/popups/index.js

```diff
@@ -0,0 +1,251 @@
+		];
+		const inactive = popups.filter(
+			( { categories, options, sitewide_default: sitewideDefault } ) =>
+				'inline' === options.placement
+					? 'never' === options.frequency
+					: 'test' !== options.frequency && ( ! sitewideDefault && ! categories.length )
+		);
+		return { test, active: activeWithSitewideDefaultFirst, inactive };
+	};
+
+	previewUrlForPopup = ( { options, id } ) => {
+		const { placement, trigger_type: triggerType } = options;
+		const previewURL =
+			'inline' === placement || 'scroll' === triggerType
+				? window && window.newspack_popups_data && window.newspack_popups_data.preview_post
+				: '/';
+		return `${ previewURL }?newspack_popups_preview_id=${ id }`;
+	};
+
+	render() {
+		const { pluginRequirements } = this.props;
+		const { popups, previewUrl } = this.state;
+		const { inline, overlay } = popups;
+		return (
+			<Fragment>
+				<WebPreview
+					url={ previewUrl }
+					renderButton={ ( { showPreview } ) => (
+						<HashRouter hashType="slash">
+							<Switch>
+								{ pluginRequirements }
+								<Route
+									path="/overlay"
+									render={ () => (
+										<PopupGroup
+											headerIcon={ <HeaderIcon /> }
+											headerText={ headerText }
+											subHeaderText={ subHeaderText }
+											tabbedNavigation={ tabbedNavigation }
+											items={ overlay }
+											buttonText={ __( 'Add new Overlay Pop-up', 'newspack' ) }
+											buttonAction="/wp-admin/post-new.php?post_type=newspack_popups_cpt"
+											setSitewideDefaultPopup={ this.setSitewideDefaultPopup }
+											setCategoriesForPopup={ this.setCategoriesForPopup }
+											updatePopup={ this.updatePopup }
+											deletePopup={ this.deletePopup }
+											emptyMessage={ __( 'No Overlay Pop-ups have been created yet.', 'newspack' ) }
+											previewPopup={ popup =>
+												this.setState( { previewUrl: this.previewUrlForPopup( popup ) }, () =>
+													showPreview()
+												)
+											}
+										/>
+									) }
+								/>
+								<Route
+									path="/inline"
+									render={ () => (
+										<PopupGroup
+											headerIcon={ <HeaderIcon /> }
+											headerText={ headerText }
+											subHeaderText={ subHeaderText }
+											tabbedNavigation={ tabbedNavigation }
+											items={ inline }
+											buttonText={ __( 'Add new Inline Pop-up', 'newspack' ) }
+											buttonAction="/wp-admin/post-new.php?post_type=newspack_popups_cpt&placement=inline"
+											setCategoriesForPopup={ this.setCategoriesForPopup }
+											updatePopup={ this.updatePopup }
+											deletePopup={ this.deletePopup }
+											emptyMessage={ __( 'No Inline Pop-ups have been created yet.', 'newspack' ) }
+											previewPopup={ popup =>
+												this.setState( { previewUrl: this.previewUrlForPopup( popup ) }, () =>
+													showPreview()
+												)
+											}
+										/>
+									) }
+								/>
```

Comment:
Suggestion: there's plenty of repetition in props between these two `PopupGroup`s. Props can be treated as object and then spread in JSX to avoid repetition:
```jsx
const sharedProps = {
  a: 42,
}
return (
  <Component {...sharedProps} />
)
```

Reply:
Good idea: https://github.com/Automattic/newspack-plugin/pull/515/commits/780f5ad61f0a43c6980c3a3c7e5e6a6d1a2e61f1